src/more-infos/more-info-sun.html

```diff
@@ -13,5 +13,11 @@
         <div class='value'>[[itemValue(item)]]</div>
       </div>
     </template>
+    <template is='dom-repeat' items="[[computeDisplayAttributes(stateObj)]]" as="attribute">
```

Comment:
Good point. I guess that we aren't adding stuff like sunspots or solar winds any time soon.